src/patterns/ec2-app.ts

```diff
@@ -64,7 +66,26 @@ export interface RestrictedAccess extends Access {
   cidrRanges: IPeer[];
 }
 
-export type AppAccess = PublicAccess | RestrictedAccess;
+/**
+ * For when you want to restrict your application's access to services running inside your VPC.
+ *
+ * Note that if your account uses Direct Connect or VPC Peering, then incoming traffic from these sources
+ * can also be allowed.
+ *
+ * Example usage:
+ * ```typescript
+ * {
+ *   scope: AccessScope.INTERNAL,
+ *   cidrRanges: [Peer.ipv4("10.0.0.0/8")]
+ * }
+ * ```
+ */
+export interface InternalAccess extends Access {
+  scope: AccessScope.INTERNAL;
+  cidrRanges: IPeer[];
+}
+
+export type AppAccess = PublicAccess | RestrictedAccess | InternalAccess;
```

Comment:
Are we still using this? If not we can maybe remove it, can't seem to spot usage in the changes.

```suggestion
```

Reply:
Looks like it is used in [`GuEc2AppProps`](https://github.com/guardian/cdk/pull/640/files#diff-161e4cde1954da9ec6ae32e3e8546e3f9befa14b6cc6d42824afe33fc7af6bc0R170)

Reply:
Oh sorry 🤦🏻 missed that you weren't newly declaring this and it already existed, ignore this!